api/src/main/java/org/openmrs/ui/framework/page/PageContext.java

```diff
@@ -189,7 +190,18 @@ public void setLocale(Locale locale) {
 
     @Override
     public void includeResource(Resource resource) {
-        resourcesToInclude.add(resource);
+    	Iterator<Resource> iterator = resourcesToInclude.iterator();
+    	while (iterator.hasNext()) {
+    	    Resource res = iterator.next();
+    	    if (res.getProviderName().equals(resource.getProviderName()) && res.getResourcePath().equals(resource.getResourcePath())) {
+    	    	if (res.getPriority() < resource.getPriority()) {

```

Comment:
What happens when the priority is the same?

Reply:
As per from here, nothing happens since the resource is already existing. So we ignore it 👍